### DISCUSSION TITLE
Q: TEMPORARY_CHANNEL_FAILURE message with unknown "short" nodes?

### DISCUSSION THREAD

**Discussion by dunkfan** [lightningnetwork/lnd#7653] [Q&A] [answered]
A routing failure report contains an info like this:
| TEMPORARY_CHANNEL_FAILURE @ 3rd hop | 124.368 | 130.547 | 12500 | 6.124 | 787404 | 843824596846575616 | LEGALIZE_MATH->0287dd->032713->023bdd->c-otto.de
How do I find out what node corresponds to "032713" the 3rd hop point?
Are such numbers hidden channels?

**Comment by guggero** [comment, marked as answer]
Those numbers starting with either 02 or 03 are node public keys. Your node tries to replace them with the node alias (e.g. LEGALIZE_MATH or c_otto.de) but sometimes due to caching reasons can't do that for all nodes on the path. So the number you see is the first part of their node identity public key (=node key). You can enter that into a network explorer such as amboss.space to find out the node. For example "032713" is https://amboss.space/node/03271338633d2d37b285dae4df40b413d8c6c791fbee7797bc5dc70812196d7d5c.
